tests/tuple-tree-generator/python-wrappers/multiple-versions/deserialize_multiple_versions.py

```diff
@@ -40,9 +41,11 @@ def test_deserialize_multiple_versions():
 
 
 def test_tagged_deserialize_multiple_versions():
-    """Tests that the custom YAML loaders can be used to deserialize multiple conflicting versions at the same time.
-    Differing from test_deserialize_multiple_versions, this test uses tagged documents to test that the default path
-    resolver installed in the `YamlLoader`s does not cause issues.
+    """Tests that the custom YAML loaders can be used to deserialize multiple
+    conflicting versions at the same time.
+    Differing from test_deserialize_multiple_versions, this test uses tagged
+    documents to test that the default path resolver installed in the
+    `YamlLoader`s does not cause issues.
```

Comment:
Seems like you're indenting at 80, but in `black` we force indentation to `100`.

Reply:
Fixed